pkg/kn/commands/channel/flags.go

```diff
@@ -120,3 +125,64 @@ func printChannelListWithNamespace(channelList *messagingv1beta1.ChannelList, op
 
 	return append(rows, others...), nil
 }
+
+// ListTypesHandlers handles printing human readable table for `kn channel list-types`
+func ListTypesHandlers(h printers.PrintHandler) {
+	channelTypesColumnDefinitions := []metav1beta1.TableColumnDefinition{
+		{Name: "Type", Type: "string", Description: "Kind / Type of the channel type", Priority: 1},
+		{Name: "Group", Type: "string", Description: "Group of the channel type", Priority: 1},
+		{Name: "Version", Type: "string", Description: "Version of the channel type", Priority: 1},
```

Comment:
We should have `Name` and `Description` fields here instead of `Grop` and `Version`.


```
➜  client git:(channel-list-types) kn channel list-types
TYPE              GROUP                   VERSION
InMemoryChannel   messaging.knative.dev   v1beta1
KafkaChannel      messaging.knative.dev   v1beta1
```
vs
```
➜  client git:(channel-list-types) kn source list-types
TYPE              NAME                                   DESCRIPTION
ApiServerSource   apiserversources.sources.knative.dev   Watch and send Kubernetes API events to addressable
ContainerSource   containersources.sources.knative.dev   Generate events by Container image and send to addressable
KafkaSource       kafkasources.sources.knative.dev       Route events from Apache Kafka Server to addressable
PingSource        pingsources.sources.knative.dev        Send periodically ping events to addressable
SinkBinding       sinkbindings.sources.knative.dev       Binding for connecting a PodSpecable to addressable
```